src/render.jl

```diff
@@ -98,15 +143,25 @@ The documentation will be available from
 `https://USER_NAME.github.io/PACKAGE_NAME/FILE_PATH.html`.
 
 """
-function save(file::String, modulename::Module; mathjax = false, include_internal = true)
+function save(file::String, modulename::Module, plmain;
+                            mathjax = false, include_internal = true, permalink::Bool = true)
     mime = MIME("text/$(strip(last(splitext(file)), '.'))")
-    save(file, mime, documentation(modulename); mathjax = mathjax, include_internal = include_internal)
+    save(file, mime, documentation(modulename), plmain; mathjax = mathjax,
+        include_internal = include_internal, permalink = permalink)
```

Comment:
I think we're going to need a `Dict{Symbol, Any}`, or something, with the potential increase in configuration settings.

```julia
function save(file::String, modulename::Module, plmain, config::Dict{Symbol, Any})
    mime = MIME("text/$(strip(last(splitext(file)), '.'))")
    save(file, mime, documentation(modulename), plmain, config)
end
```

Not with this PR though, we can change it later on.